src/streaming/types/asset.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Create Asset Clip
+ */
```

Comment:
Could you please remove this block as it might cause issues with the documentation generation.